src/main/kotlin/org/wycliffeassociates/otter/common/data/audioplugin/AudioPluginData.kt

```diff
@@ -10,5 +10,5 @@ data class AudioPluginData(
         var canRecord: Boolean,
         var executable: String,
         var args: List<String>,
-        var pluginFile: File
+        var pluginFile: File?
```

Comment:
I know this is a consequence of the UI generated plugins.

That said, should we prefer that the UI actually writes out a plugin file rather than make this field nullable?

Reply:
I think I was working off a previous discussion where we decided against writing out a plugin file since it would end up in the private app directory that users wouldn't need to see/touch.